homeassistant/components/sensor/rfxtrx.py

```diff
@@ -110,9 +111,9 @@ def __str__(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self.data_type:
+        if self.event:
             return self.event.values[self.data_type]
-        return None
+        return STATE_UNKNOWN
```

Comment:
Ok, I will change it back to returning None.

Before this pr the state function never returned None, "if self.data_type" was always True